packages/db/tests/translate/test-granules.js

```diff
@@ -615,7 +616,7 @@ test('translatePostgresGranuleToApiGranule handles granule with no associated ex
   );
 });
 
-test('translateApiGranuleToPostgresGranule converts API granule to Postgres', async (t) => {
+test('translateApiGranuleToPostgresGranuleWithoutNilsRemoved converts API granule to Postgres', async (t) => {
```

Comment:
Gven they're both exported, should we have a unit test for `translateApiGranuleToPostgresGranule converts API granule to Postgres`  as well?  The test below is only asserting that the nills/undefined values are removed.

Reply:
Yup. I was think that would be duplicative because `translateApiGranuleToPostgresGranule ` just calls `translateApiGranuleToPostgresGranuleWithoutNilsRemoved ` but the tests should be implementation-agnostic. Also added a test to explicitly assert that nils are not removed where appropriate:

https://github.com/nasa/cumulus/pull/3082/commits/37576e12cc6edef9d5c542971d838d7661b32e36